Src/Private/Get-AbrFgtSystem.ps1

```diff
@@ -96,7 +96,7 @@ function Get-AbrFgtSystem {
             }
 
             if ($info -and $settings -and $InfoLevel.System -ge 1) {
-                Section -Style Heading3 'GUI Settings' {
+                Section -Style Heading3 'Feature GUI visibility' {
```

Comment:
change also TableParams name ?

Reply:
and only to make also modification line 130 (TableParams = @{
                        Name         = "Settings"